tools/mo/unit_tests/mo/ops/roialign_test.py

```diff
@@ -0,0 +1,140 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import unittest
+from openvino.tools.mo.front.common.partial_infer.utils import int64_array, float32_array
+
+import numpy as np
+from openvino.tools.mo.graph.graph import Node
+from openvino.tools.mo.ops.roialign import ROIAlign
+from unit_tests.utils.graph import build_graph
+
+
+class TestROIAlignOps(unittest.TestCase):
```

Comment:
There are additional tests for MO ONNX ROIAlign:
https://github.com/openvinotoolkit/openvino/blob/252c72146b8c7ec054bc27f001bbdef52c783f51/tests/layer_tests/onnx_tests/test_roi_align.py#L107-L109
Could you enable it (`@pytest.mark.precommit`) at least for one CI cycle?
Also you can update these tests if needed, and please ensure they won't go by default MO path, so with ONNX FE (onnx importer).